pkg/registry/servicecatalog/broker/strategy.go

```diff
@@ -111,7 +111,7 @@ func (brokerRESTStrategy) PrepareForUpdate(ctx kapi.Context, new, old runtime.Ob
 	if !ok {
 		glog.Fatal("received a non-broker object to update from")
 	}
-	newBroker.Spec = oldBroker.Spec
+
```

Comment:
Not sure I grok this. Isn't this a bug that should probably be fixed in its own PR?